ptf/saidebugcounters.py

```diff
@@ -500,44 +525,43 @@ def testPortDebugCounter(self,
         if not isinstance(dc_oid, list):
             dc_oid = [dc_oid]
 
+        port = self.port0
+
```

Comment:
Unfortunately, pylint throws "Too many arguments" warning when using more than 5 arguments. Every call to this function used port = self.port0 as an argument. This is a work-around for this issue.

Reply:
Ok, I thought that pylint is used here and there. I will remove the workaround, thanks.